pkg/controller/machine/machine.go

```diff
@@ -107,17 +107,19 @@ type Controller struct {
 }
 
 type NodeSettings struct {
-	// Translates to --cluster-dns on the kubelet
+	// Translates to --cluster-dns on the kubelet.
 	ClusterDNSIPs []net.IP
 	// If set, this proxy will be configured on all nodes.
 	HTTPProxy string
-	// If set this will be set as NO_PROXY on the node
+	// If set this will be set as NO_PROXY on the node.
 	NoProxy string
-	// If set, this image registry will be used for pulling all required images on the node
+	// If set, this image registry will be used for pulling all required images on the node.
```

Comment:
Could you fix this comment? I screwed that up in the initial PR :(

`If set, those registries will be configured as insecure on the container runtime.`

Reply:
Done. Will rush in with next push.